src/enforcer.js

```diff
@@ -541,13 +637,14 @@ export class TrustedTypesEnforcer {
 
   /**
    * Wraps a setter with the enforcement wrapper.
-   * @param {!Object} object The object of the to-be-wrapped property.
+   * @param {Object} object The object of the to-be-wrapped property.
```

Comment:
Keep it as non-nullable (and skip calling with null as an object).

Reply:
Done.